### DISCUSSION TITLE
Custom Word Wrapping

### DISCUSSION THREAD

**Discussion by RedAlbatross** [jonasthewolf/gsn2x#315] [Ideas]
Many thanks for the recent update and allowing the GSN Standard example to work. This is going to prove a really useful tool for me and thank you for your quick responses. 

Currently, long descriptions causes the boxes to be quite wide:
![image](https://github.com/jonasthewolf/gsn2x/assets/144147111/2f4019d5-4fd0-40cf-b995-08a7bac4ac09)

Whereas in the GSN standard, they are able to keep them more compact:
![image](https://github.com/jonasthewolf/gsn2x/assets/144147111/966ef605-7a3a-4bca-9476-3d6e35506f8a)

I was wondering if you could add an option, perhaps through the style sheet, to define the word wrapping length?

I would try to edit the svg file, but that would require changing the size of the boxes and would affect their placement.

Kind regards,

**Comment by jonasthewolf** [comment]
Hi @RedAlbatross,

it is possible to introduce manual line breaks if I remember correctly.

Example:

```yaml
G1:
  text: |
    this
    should
    break
    like
    here
```

My line breaks are just to prevent too large boxes.

**Comment by RedAlbatross** [comment]
@jonasthewolf 
That works perfectly, thank you.
